test/middleware.ts

```diff
@@ -0,0 +1,11 @@
+import {
+  Context, Middleware, Next, Response,
+} from 'koa';
```

Comment:
Not sure why eslint wants it like this?

Reply:
Narrowed it down to https://github.com/airbnb/javascript/blob/2e3adc98c784be7fa63c7a05271f7aa86c9d6bc7/packages/eslint-config-airbnb-base/rules/style.js#L399. Something to look at when we create a Libero config.

Reply:
I agree with https://github.com/airbnb/javascript/issues/1051#issuecomment-244526891